.github/workflows/daily-aarch64-development.yml

```diff
@@ -0,0 +1,168 @@
+        run: |
+          zip -R artifacts-jvm-aarch64-${{ matrix.category }}.zip '*-reports/*'
+
+      - name: Archive artifacts
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: artifacts-linux-jvm-aarch64-${{ matrix.category }}
+          path: artifacts-jvm-aarch64-${{ matrix.category }}.zip
+
+      - name: Delete Local Artifacts From Cache
+        shell: bash
+        run: rm -rf ~/.m2/repository/org/acme
+  report-aarch64:
+    name: Report - aarch64
+    runs-on: ubuntu-latest
+    needs: [ aarch64-linux-build-jvm-latest, aarch64-linux-build-native-latest ]
+    if: "always() && github.repository == 'quarkusio/quarkus-quickstarts'"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'development' }}
+      - uses: quarkusio/report-status-in-issue@main
+        with:
+          github-token: ${{ secrets.GITHUB_API_TOKEN }}
+          status: ${{ (needs.aarch64-linux-build-jvm-latest.result == 'success' && needs.aarch64-linux-build-native-latest.result == 'success') && 'success' || 'failure' }}
+          issue-repository: quarkusio/quarkus
+          issue-number: 6588
+
+  aarch64-linux-build-native-latest:
+    name: Daily - Aarch64 - Linux - Native build - Latest Version
+    runs-on: ubuntu-24.04-arm
+    needs: [ generate-json-matrix ]
+    if: "github.repository == 'quarkusio/quarkus-quickstarts' || github.event_name == 'workflow_dispatch'"
+    strategy:
+      fail-fast: false
+      max-parallel: 3
+      matrix: ${{fromJson(needs.generate-json-matrix.outputs.matrix) }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'workflow_dispatch' && github.event.ref || 'development' }}
+      - name: Reclaim Disk Space
+        run: .github/ci-prerequisites.sh
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "date=$(/bin/date -u "+%Y-%m")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Cache Maven Repository
+        id: cache-maven
+        uses: actions/cache@v4
+        with:
+          path: ~/.m2/repository
+          key: q2maven-native-aarch64-${{ steps.get-date.outputs.date }}
+
+      - name: Install JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+          check-latest: true
+          cache: 'maven'
+
+      - name: Build Quarkus main
+        run: |
+          git clone https://github.com/quarkusio/quarkus.git
+          cd quarkus
+          ./mvnw -T1C -e -B --settings .github/mvn-settings.xml clean install -Dquickly-ci
+
+      - name: Test Quickstarts in Native mode on aarch64
+        run: |
+          ./mvnw -e -B --settings .github/mvn-settings.xml clean install --fail-at-end -Dnative -Dstart-containers \
+            -Dquarkus.native.container-build=true -Daarch64 -am -pl "${{ matrix.test-modules }}"
```

Comment:
My understanding is that it will also run the JVM tests in JVM mode so I don't think we need the first JVM job above.

Now maybe you absolutely want to split the JVM coverage?